web/docs/faqs.md

```diff
@@ -173,54 +172,59 @@ data analysis.
 
 Tenzir does not rely on a third-party database.
 
-Tenzir nodes include a light-weight storage engine on top of partitioned Feather
-or Parquet files, accessible via the [`import`](operators/import.md) and
-[`export`](operators/export.md) operators. The engine comes with a
-catalog that tracks meta data and a thin layer of sketches to accelerate
-queries.
+Tenzir nodes include a light-weight storage engine on top of Feather or Parquet
+files, accessible via the [`import`](operators/import.md) and
+[`export`](operators/export.md) operators.
+
+The storage engine comes with a catalog that tracks schema meta data and a thin
+layer of sketches to accelerate queries.
```

Comment:
```suggestion
layer of indexing to accelerate queries.
```

Sketching is a very technical term that people without database skills will not know.